arch/arm/boot/dts/overlays/vc4-kms-v3d-pi5-overlay.dts

```diff
@@ -3,7 +3,7 @@
 #include "cma-overlay.dts"
 
 &frag0 {
-	size = <((320-4)*1024*1024)>;
+	size = <(64*1024*1024)>;
```

Comment:
The fact that this is a multiple of 64MB means it will not be automatically capped to the platform-limit, but this is way below all the limits so not an issue.

Reply:
Yes - I spent a while reading the firmware code and came to same conclusion.